spec/models/activity_spec.rb

```diff
@@ -231,4 +231,65 @@
       expect(activity.active_course?).to eq(false)
     end
   end
+
```

Comment:
We currently do not test that the credits are not changed if there is no duration in hours. As we use that as the primary criteria in the callback in the programme activity, would be good to test that behaviour

Reply:
I've added a test for this on line 246